bootstrap.d/dev-libs.yml

```diff
@@ -1083,24 +1083,20 @@ packages:
     architecture: '@OPTION:arch@'
     source:
       subdir: ports
-      git: 'https://gitlab.gnome.org/GNOME/libxslt.git'
-      tag: 'v1.1.34'
-      version: '1.1.34'
-      tools_required:
-        - host-autoconf-v2.69
-        - host-automake-v1.15
-        - host-libtool
-        - host-pkg-config
+      url: 'https://download.gnome.org/sources/libxslt/1.1/libxslt-1.1.38.tar.xz'
+      format: 'tar.xz'
+      checksum: blake2b:9397be8b223bd68b326b954fe6a749a269e969b4669747678715728f553916c23475232fc2894a5e4230e15b031981423adeb8f83407f4db8075a784c7e6a55b
+      extract_path: 'libxslt-1.1.38'
+      version: '1.1.38'
```

Comment:
Core libs should continue to use VCS sources.

Reply:
Converted to VCS and CMake.